fitbit-react/src/Pages/Wishlist/WishList.js

```diff
@@ -12,7 +13,7 @@ function WishList() {
         <h3 class="h2 centered-text">My WishList</h3>
```

Comment:
Suggestion: I think you forgot to change `class` to  `className`

Reply:
Done